stream-transformation-test/stream-transformation-it/pom.xml

```diff
@@ -12,4 +12,164 @@
 
     <artifactId>stream-transformation-it</artifactId>
 
+    <dependencies>
+
+        <dependency>
+            <groupId>com.h2database</groupId>
+            <artifactId>h2</artifactId>
+            <scope>test</scope>
+            <version>${com.h2database.version}</version>
+        </dependency>
+
+        <dependency>
+            <groupId>uk.gov.justice.utils</groupId>
```

Comment:
if we set this in managed dependencies then we shouldn't need to do exclusions each time (I think)